AI/src/main/java/dk/gruppeseks/bodtrd/ai/AIProvider.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package dk.gruppeseks.bodtrd.ai;
+
+import dk.gruppeseks.bodtrd.common.data.entityelements.Position;
+import dk.gruppeseks.bodtrd.common.services.AISPI;
+
+/**
+ *
+ * @author frede
+ */
+public class AIProvider implements AISPI
+{
+    @Override
+    public Position[] getPath(Position start, Position goal)
+    {
+        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
```

Comment:
can you exclude this component until it does something?
you might later find that this wasnt the approach you wanted and then we just have shit to remove in different files